sources/packages/backend/libs/integrations/src/services/sfas/sims-to-sfas.service.ts

```diff
@@ -141,7 +141,7 @@ export class SIMSToSFASService {
         .addSelect("student.id", "studentId")
         .addSelect("programYear.programYear", "programYear")
         .addSelect(
-          "COALESCE(offering.offeringIntensity, (application.data->>'howWillYouBeAttendingTheProgram')::sims.offering_intensity)",
+          "COALESCE(offering.offeringIntensity, application.offeringIntensity)",
```

Comment:
Minor, similar to this one, the `application.offeringIntensity` can now be used directly.